content/zh/docs/tasks/job/parallel-processing-expansion.md

```diff
@@ -368,6 +368,23 @@ Kubernetes accepts and runs the Jobs you created.
 -->
 Kubernets 接收清单文件并执行你所创建的 Job。
 
+<!--
+### Clean up {#cleanup-2}
+```shell
+# Remove the Jobs you created
+# Your cluster automatically cleans up their Pods
+kubectl delete job -l jobgroup=jobexample
+```
+-->
+### 清理
```

Comment:
do NOT throw away the explicit anchor ID.

Reply:
suggestion applied

Reply:
the commits have been squashed into one commit

Reply:
can this commit merged ？